channels/us.m3u

```diff
@@ -151,17 +151,17 @@ https://a.jsrdn.com/broadcast/542cb2ce3c/+0000/c.m3u8
 http://31.220.41.88:8081/live/us-cinemaxaction.stream/playlist.m3u8
 #EXTINF:-1 tvg-id="LocalNowAdrenalineRush.us" tvg-name="Local Now Adrenaline Rush" tvg-country="US" tvg-language="English" tvg-logo="https://f9q4g5j6.ssl.hwcdn.net/601341747e82bb6a262f276b" group-title="Entertainment",Adrenaline Rush
 https://linear-133.frequency.stream/dist/localnow/133/hls/master/playlist.m3u8
-#EXTINF:-1 tvg-id="AdultSwim.us" tvg-name="Adult Swim" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/ixywxhs.jpg" group-title="Entertainment",Adult Swim (1080p) [Not 24/7]
+#EXTINF:-1 tvg-id="AdultSwim.us" tvg-name="Adult Swim" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/ixywxhs.jpg" group-title="Animation",Adult Swim (1080p) [Not 24/7]
```

Comment:
Adult Swim as a channel itself shows variety of content. Entertainment was right category